src/java.base/share/classes/java/lang/invoke/MethodType.java

```diff
@@ -1168,7 +1168,8 @@ Invokers invokers() {
      * <p>
      * Note that it is possible to encounter method types which cannot be
```

Comment:
BTW this can be converted to `@apiNote` as well.

Reply:
Done.